airbase-policy/src/main/resources/checkstyle/airbase-checks.xml

```diff
@@ -70,6 +70,10 @@
         <property name="format" value="^(?!import ).*Math\.toIntExact" />
         <property name="message" value="Math.toIntExact should only be used with static imports" />
     </module>
+    <module name="RegexpSingleline">
+        <property name="format" value="^(?!import ).*String\.format" />
```

Comment:
This change could be too invasive, especially because another rule exists:
https://github.com/airlift/airbase/blob/fb4d29a27361c6a0a7f11bce8c0b399960bdd62d/airbase-policy/src/main/resources/checkstyle/airbase-checks.xml#L57

Could you please make it a little bit more strict, so that `MyBetter123_String` or `my.Package.String` could have `format` function as well.
So you have to filter out only
`java.lang.String.format` and
`String.format` with `[^a-zA-Z_$0-9])String\.format`